### PR TITLE
sysdig 0.26.2

### PR DESCRIPTION
**Pull request by Moisan**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
-- Found zlib: include: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include, lib: /usr/lib/libz.dylib
CMake Error at CMakeLists.txt:252 (message):
  Couldn't find system tbb
```

**Comment by Moisan** [comment]
The error is now
```
/Library/Developer/CommandLineTools/usr/include/c++/v1/cmath:325:9: error: no member named 'isunordered' in the global namespace
using ::isunordered;
      ~~^
```
I tried `ENV.delete("SDKROOT")` and `ENV.delete("CPATH")` without success.

Upstream also uses the `coreutils` formula in their macOS build, adding it as a dependency didn't help either. Adding the "help wanted" label because I don't know what else to try.

**Comment by equal-l2** [comment]
In my environment, the build error differ:

```
/usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++   -Wall -ggdb -std=c++0x -O3 -fno-strict-aliasing -DNDEBUG -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names -pagezero_size 10000 -image_base 100000000 CMakeFiles/csysdig.dir/fields_info.cpp.o CMakeFiles/csysdig.dir/csysdig.cpp.o  -o csysdig ../libsinsp/libsinsp.a /usr/lib/libncurses.dylib /usr/lib/libform.dylib ../libscap/libscap.a /usr/lib/libz.dylib /usr/local/lib/libjsoncpp.dylib /usr/local/opt/openssl/lib/libssl.dylib /usr/local/opt/openssl/lib/libcrypto.dylib /usr/local/lib/libluajit.dylib -ldl -lpthread
Undefined symbols for architecture x86_64:
  "tbb::internal::NFS_Allocate(unsigned long, unsigned long, void*)", referenced from:
      sinsp::sinsp() in libsinsp.a(sinsp.cpp.o)
      tbb::strict_ppl::concurrent_queue<std::__1::shared_ptr<sinsp_evt>, tbb::cache_aligned_allocator<std::__1::shared_ptr<sinsp_evt> > >::allocate_block(unsigned long) in libsinsp.a(sinsp.cpp.o)
  "tbb::internal::throw_exception_v4(tbb::internal::exception_id)", referenced from:
      tbb::strict_ppl::concurrent_queue<std::__1::shared_ptr<sinsp_evt>, tbb::cache_aligned_allocator<std::__1::shared_ptr<sinsp_evt> > >::allocate_block(unsigned long) in libsinsp.a(sinsp.cpp.o)
      tbb::strict_ppl::internal::micro_queue<std::__1::shared_ptr<sinsp_evt> >::spin_wait_until_my_turn(tbb::atomic<unsigned long>&, unsigned long, tbb::strict_ppl::internal::concurrent_queue_rep_base&) const in libsinsp.a(container.cpp.o)
  "tbb::internal::NFS_Free(void*)", referenced from:
      tbb::strict_ppl::concurrent_queue<std::__1::shared_ptr<sinsp_evt>, tbb::cache_aligned_allocator<std::__1::shared_ptr<sinsp_evt> > >::deallocate_block(void*, unsigned long) in libsinsp.a(sinsp.cpp.o)
      tbb::strict_ppl::internal::concurrent_queue_base_v3<std::__1::shared_ptr<sinsp_evt> >::~concurrent_queue_base_v3() in libsinsp.a(sinsp.cpp.o)
  "_curl_easy_cleanup", referenced from:
      libsinsp::container_engine::docker_async_source::free_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_easy_getinfo", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_easy_init", referenced from:
      libsinsp::container_engine::docker_async_source::init_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_easy_setopt", referenced from:
      libsinsp::container_engine::docker_async_source::init_docker_conn() in libsinsp.a(docker_linux.cpp.o)
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_add_handle", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_cleanup", referenced from:
      libsinsp::container_engine::docker_async_source::free_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_init", referenced from:
      libsinsp::container_engine::docker_async_source::init_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_perform", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_remove_handle", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_setopt", referenced from:
      libsinsp::container_engine::docker_async_source::init_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_wait", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [userspace/sysdig/csysdig] Error 1
make[1]: *** [userspace/sysdig/CMakeFiles/csysdig.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[100%] Linking CXX executable sysdig
cd /tmp/sysdig-20190707-79686-1gpptz9/sysdig-0.26.1/build/userspace/sysdig && /usr/local/Cellar/cmake/3.14.5/bin/cmake -E cmake_link_script CMakeFiles/sysdig.dir/link.txt --verbose=1
/usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++   -Wall -ggdb -std=c++0x -O3 -fno-strict-aliasing -DNDEBUG -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names -pagezero_size 10000 -image_base 100000000 CMakeFiles/sysdig.dir/fields_info.cpp.o CMakeFiles/sysdig.dir/sysdig.cpp.o  -o sysdig ../libsinsp/libsinsp.a ../libscap/libscap.a /usr/lib/libz.dylib /usr/local/lib/libjsoncpp.dylib /usr/local/opt/openssl/lib/libssl.dylib /usr/local/opt/openssl/lib/libcrypto.dylib /usr/local/lib/libluajit.dylib -ldl -lpthread
Undefined symbols for architecture x86_64:
  "tbb::internal::NFS_Allocate(unsigned long, unsigned long, void*)", referenced from:
      sinsp::sinsp() in libsinsp.a(sinsp.cpp.o)
      tbb::strict_ppl::concurrent_queue<std::__1::shared_ptr<sinsp_evt>, tbb::cache_aligned_allocator<std::__1::shared_ptr<sinsp_evt> > >::allocate_block(unsigned long) in libsinsp.a(sinsp.cpp.o)
  "tbb::internal::throw_exception_v4(tbb::internal::exception_id)", referenced from:
      tbb::strict_ppl::concurrent_queue<std::__1::shared_ptr<sinsp_evt>, tbb::cache_aligned_allocator<std::__1::shared_ptr<sinsp_evt> > >::allocate_block(unsigned long) in libsinsp.a(sinsp.cpp.o)
      tbb::strict_ppl::internal::micro_queue<std::__1::shared_ptr<sinsp_evt> >::spin_wait_until_my_turn(tbb::atomic<unsigned long>&, unsigned long, tbb::strict_ppl::internal::concurrent_queue_rep_base&) const in libsinsp.a(container.cpp.o)
  "tbb::internal::NFS_Free(void*)", referenced from:
      tbb::strict_ppl::concurrent_queue<std::__1::shared_ptr<sinsp_evt>, tbb::cache_aligned_allocator<std::__1::shared_ptr<sinsp_evt> > >::deallocate_block(void*, unsigned long) in libsinsp.a(sinsp.cpp.o)
      tbb::strict_ppl::internal::concurrent_queue_base_v3<std::__1::shared_ptr<sinsp_evt> >::~concurrent_queue_base_v3() in libsinsp.a(sinsp.cpp.o)
  "_curl_easy_cleanup", referenced from:
      libsinsp::container_engine::docker_async_source::free_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_easy_getinfo", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_easy_init", referenced from:
      libsinsp::container_engine::docker_async_source::init_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_easy_setopt", referenced from:
      libsinsp::container_engine::docker_async_source::init_docker_conn() in libsinsp.a(docker_linux.cpp.o)
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_add_handle", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_cleanup", referenced from:
      libsinsp::container_engine::docker_async_source::free_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_init", referenced from:
      libsinsp::container_engine::docker_async_source::init_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_perform", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_remove_handle", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_setopt", referenced from:
      libsinsp::container_engine::docker_async_source::init_docker_conn() in libsinsp.a(docker_linux.cpp.o)
  "_curl_multi_wait", referenced from:
      libsinsp::container_engine::docker_async_source::get_docker(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libsinsp.a(docker_linux.cpp.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

According to bisect, this started to occur since https://github.com/draios/sysdig/commit/199d5a68ee35b76b067bcdf14f205a9cfc33133d

**Comment by adalton** [comment]
sysdig-0.26.2 was released on August 6, and includes a change that should address the build issues described above:

https://github.com/draios/sysdig/releases/tag/0.26.2
https://github.com/draios/sysdig/pull/1415

If it'd be more helpful to open a new PR for that version, please let me know.

**Comment by SMillerDev** [comment]
@Moisan wanna update this?

**Comment by EricFromCanada** [comment]
For me, 0.26.2 builds and passes on 10.12.

File: Formula/sysdig.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Sysdig < Formula
   desc "System-level exploration and troubleshooting tool"
   homepage "https://www.sysdig.org/"
-  url "https://github.com/draios/sysdig/archive/0.24.2.tar.gz"
-  sha256 "cd925afd2fb0a26728611666e017d480afd49158c2d70714c7461a97c8820807"
+  url "https://github.com/draios/sysdig/archive/0.26.1.tar.gz"
```
Comment (EricFromCanada) [inline Formula/sysdig.rb]:
```suggestion
  url "https://github.com/draios/sysdig/archive/0.26.2.tar.gz"
```

File: Formula/sysdig.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+  sha256 "4bc59f4e958c9b971ecb4d6fe98fe0be27a185b87106c76aaf4614ab3ef7129e"
```
Comment (EricFromCanada) [inline Formula/sysdig.rb]:
```suggestion
  sha256 "6f4f5b7b187c3774b6c374c1728b3f905ac18a945bde15151dcfb24c79abb441"
```

**Comment by Moisan** [comment]
Thank you @adalton and @EricFromCanada for the help!

**Pull request closed without merging**